docs/rules/no-anonymous-default-export.md

```diff
@@ -0,0 +1,51 @@
+# no-anonymous-default-export
+
+Reports if a module's default export is unnamed. This includes several types of unnamed data types; literals, object expressions, anonymous functions, arrow functions, and anonymous class declarations.
```

Comment:
I would also enforce this on arrays, which I for instance sometimes use to create a fixtures file.
Also, I think it would make sense to also handle template literals if we intend to handle pretty much any value ?